src/utils/RNPxlSearch.cpp

```diff
@@ -1999,7 +1999,7 @@ class RNPxlSearch :
         double precursor_mz = precursor[0].getMZ();
 
         // map (corrected) precursor mass to spectra
-        for (auto i : precursor_isotopes)
+        for (int i : precursor_isotopes)

```

Comment:
I changed it to make it easier readable. Auto is great for long typenames but not really needed for ints